training/distributed_training/pytorch/model_parallel/GPT2_huggingface_trainer/submit_smp_trainer.ipynb

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        prescaled_batch = True
</pre><p>When pipeline degree &lt;=1, we do not enable prescaled batch, </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3577/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Thanks for the catch!</p>